src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
```

Comment:
Máš point, ale je fakt, že ty proměnné snad vůbec nikdo nepoužívá dobře. Zrovna na Darujme v tom děláš bordel ty, Honza, ale i já, takže ti to těžko můžu omlátit o hlavu, když to sám neovládám. Že v tom děláš chyby byl teď nejdříve jen můj odhad, protože mi to bylo úplně jasné. :D